flake.nix

```diff
@@ -10,16 +10,18 @@
       inputs.flake-utils.follows = "flake-utils";
     };
     zaozi.url = "github:sequencer/zaozi";
+    treefmt-nix.url = "github:numtide/treefmt-nix";
   };
 
-  outputs = { self, nixpkgs, mill-ivy-fetcher, flake-utils, zaozi }@inputs:
+  outputs = { self, nixpkgs, mill-ivy-fetcher, flake-utils, zaozi, treefmt-nix }@inputs:
     let
       overlay = import ./nix/overlay.nix { inherit self; };
     in
     flake-utils.lib.eachDefaultSystem
       (system:
         let
           pkgs = import nixpkgs { inherit system; overlays = [ mill-ivy-fetcher.overlays.default zaozi.overlays.default overlay ]; };
+          treefmtEval = treefmt-nix.lib.evalModule pkgs ./treefmt.nix;
```

Comment:
I think we can just use:

```nix
        treefmtEval = treefmt-nix.lib.evalModule pkgs {
          projectRootFile = "flake.nix";
          settings.verbose = 1;
          programs.nixfmt.enable = true;
        };
```

since we don't want to use treefmt for all language yet.

Reply:
https://treefmt.com/latest/guides/unmatched-formatters/#enforcing-strict-matching I apply this suggestion.